app/src/app.py

```diff
@@ -1,14 +1,26 @@
 from flask import Flask, render_template
 from models import db, User
```

Comment:
Relative imports, from local files from within the project, need to have a `.` infront of them. So it should be `.models`. External packages, like `flask`, do not need to have a `.` infront of them.